synfig-studio/src/gui/workarea.cpp

```diff
@@ -2185,15 +2194,22 @@ studio::WorkArea::set_cursor(const Glib::RefPtr<Gdk::Cursor> &x)
 	//!Check if the window we want draw is ready
 	Glib::RefPtr<Gdk::Window> draw_area_window = drawing_area->get_window();
 	if (!draw_area_window) return;
-	draw_area_window->set_cursor(x);
+
+	// do not change cursor in drag window mode
+	if (drag_mode == DRAG_WINDOW) {
+		previous_cursor = x;
```

Comment:
this is necessary to prevent changing the cursor while dragging the window (for example by pressing alt to scale).  So we simply store the request cursor and restore it when dragging is finished